test/unit_tests/arkode/CXX_parallel/ark_test_heat2D_mri_0.out

```diff
@@ -0,0 +1,52 @@
+
+2D Heat PDE test problem:
+   nprocs = 2
+   nx = 60
+   ny = 120
+   kx = 0.5
+   ky = 0.75
+   rtol = 1e-05
+   atol = 1e-10
+   nxl (proc 0) = 30
+   nyl (proc 0) = 120
+   Nonlinear implicit solver
+
+
+ARKStep Solver Statistics:
+   Internal solver steps = 1000
+   Total RHS evals:  Fe = 0,  Fi = 2769
+   Total linear solver setups = 50
+   Total linear iterations = 6625
+   Total number of Jacobian-vector products = 6625
+   Total number of Preconditioner setups = 17
+   Total number of Preconditioner solves = 6625
+   Total number of linear solver convergence failures = 0
+   Total number of Newton iterations = 1768
+   Total number of nonlinear solver convergence failures = 0
+
+MRIStep Solver Statistics:
+   Internal solver steps = 1000
+   Total RHS evals:  Fe = 2769
+   Total linear solver setups = 50
+   Total linear iterations = 6241
+   Total number of Jacobian-vector products = 6241
+   Total number of Preconditioner setups = 50
+   Total number of Preconditioner solves = 6241
+   Total number of linear solver convergence failures = 0
+   Total number of Newton iterations = 1768
+   Total number of nonlinear solver convergence failures = 0
+
+Comparing Solver Statistics:
+  RHS evals error: 2769 vs 2769
```

Comment:
I fixed the comparison in the test code but forgot to update the output file afterwards. Since this test is treated as a pass/fail test based on the return value the output files were not diffed so the CI missed this as well (good catch!).